### PR TITLE
Aligning TestApp UI with other platforms

### PR DESCRIPTION
**Pull request by MatkovIvan**
## Before
device-2018-08-17-163337  device-2018-08-17-163359

## After
device-2018-08-17-163202  device-2018-08-17-163222

### CODE REVIEW

File: TestApp/index.js [hunk 1]
```diff
@@ -1,30 +1,4 @@
...
-  Push: { screen: PushScreen },
-  AppCenter: { screen: AppCenterScreen }
-});
-
-AppRegistry.registerComponent('TestApp', () => TestApp);
-
-// Ignore react-navigation 'isMounted' deprecation message (https://github.com/react-navigation/react-navigation/issues/3956)
-YellowBox.ignoreWarnings(['Warning: isMounted(...) is deprecated', 'Module RCTImageLoader']);
-
-export default TestApp;
```
Comment (bmourat) [inline TestApp/index.js]: `export default App;`

This line should stay

File: TestApp/.eslintrc [hunk 2]
```diff
@@ -9,6 +9,7 @@
     ],
     "extends": "airbnb",
     "rules": {
+        "global-require": 0,
```
Comment (guperrot) [inline TestApp/.eslintrc]: @dhei thoughts? https://eslint.org/docs/rules/global-require

**Comment by guperrot** [comment]
Updated screens compared to https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/376#issue-209162907

device-2018-08-17-180519
device-2018-08-17-180741
device-2018-08-17-180843

**Comment by MatkovIvan** [comment]
@guperrot Thanks for jump in!

File: TestApp/.eslintrc (hunk 2 above)
Comment (dhei) [reply TestApp/.eslintrc]: It's better to keep this rule and make exceptions individually by `/* eslint-disable global-require */` comments.

File: TestApp/app/screens/AnalyticsScreen.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+import { Image, View, Text, Switch, SectionList, TouchableOpacity } from 'react-native';
+import Toast from 'react-native-simple-toast';
+
+import Analytics from 'appcenter-analytics';
+
+import SharedStyles from '../SharedStyles';
+
+export default class AnalyticsScreen extends Component {
+  static navigationOptions = {
+    tabBarIcon: () => <Image style={{ width: 24, height: 24 }} source={require('../assets/analytics.png')} />,
```
Comment (dhei) [inline TestApp/app/screens/AnalyticsScreen.js]: See my comment in .eslintrc, let's disable global-require rule here. 
```
source={require('../assets/analytics.png')} // eslint-disable global-require
```

File: TestApp/app/screens/CrashesScreen.js [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+import Crashes from 'appcenter-crashes';
+
+import { FooClass } from '../../js/FooClass';
+import AttachmentsProvider from '../AttachmentsProvider';
+import SharedStyles from '../SharedStyles';
+
+export default class CrashesScreen extends Component {
+  static navigationOptions = {
+    // eslint-disable-line global-require
```
Comment (dhei) [inline TestApp/app/screens/CrashesScreen.js]: Remove this comment and do the same as https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/376/files#r211334399

**Pull request merged**